feature/gnoi/system/tests/supervisor_switchover_test/supervisor_switchover_test.go

```diff
@@ -190,6 +190,13 @@ func findComponentsByType(t *testing.T, dut *ondatra.DUTDevice, cType telemetry.
 func findStandbyRP(t *testing.T, dut *ondatra.DUTDevice, supervisors []string) (string, string) {
 	var activeRP, standbyRP string
 	for _, supervisor := range supervisors {
+		watch := dut.Telemetry().Component(supervisor).RedundantRole().Watch(
+			t, 5*time.Minute, func(val *telemetry.QualifiedE_PlatformTypes_ComponentRedundantRole) bool {
+				return val.IsPresent()
+			})
+		if val, ok := watch.Await(t); !ok {
+			t.Fatalf("DUT did not reach target state within %v: got %v", 5*time.Minute, val)
```

Comment:
nit, target/stable ? because your changes check for the existence of any value, not some expected value, right?

Reply:
Right, it checks the existence of the value.